src/site/content/en/blog/hands-on-portals/index.md

```diff
@@ -0,0 +1,203 @@
+
+<img class="w-screenshot" src="portal-preview-demo.gif" alt="A gif of preview portal style demo">
+
+```javascript
+// Adding some styles with transitions
+const style = document.createElement('style');
+const initialScale = 0.4;
+style.innerHTML = `
+  portal {
+    position:fixed;
+    width: 100%;
+    height: 100%;
+    opacity: 0;
+    transition:
+      transform 0.4s,
+      bottom 0.7s,
+      left 0.7s,
+      opacity 1.0s;
+    box-shadow: 0 0 20px 10px #999;
+    transform: scale(${initialScale});
+    bottom: calc(20px + 50% * ${initialScale} - 50%);
+    left: calc(20px + 50% * ${initialScale} - 50%);
+    z-index: 10000;
+  }
+  .portal-reveal {
+    transform: scale(1.0);
+    bottom: 0px;
+    left: 0px;
+  }
+  .fade-in {
+    opacity: 1.0;
+  }
+`;
+const portal = document.createElement('portal');
+// Let’s navigate into the WICG Portals spec page
+portal.src = 'https://wicg.github.io/portals/';
+portal.addEventListener('click', evt => {
+  // Animate the portal once user interacts
+  portal.classList.add('portal-reveal');
+});
+portal.addEventListener('transitionend', evt => {
+  if (evt.propertyName == 'bottom') {
+    // Activate the portal once the transition has completed
+    portal.activate();
+  }
+});
+document.body.append(style, portal);
+
+// Waiting for the page to load.
+// using setTimeout is a suboptimal way and it’s best to fade-in
+// when receiving a load complete message from the portal via postMessage
+setTimeout(_ => portal.classList.add('fade-in'), 2000);
+```
+
+It is also easy to do feature detection to progressively enhance a website using Portals.
+
+```javascript
+if ('HTMLPortalElement' in window) {
+  // If this is a platform that have Portals...
+  const portal = document.createElement('portal');
+  ...
+}
+```
+
+If you want to quickly experience what Portals feel like, try using [uskay-portals-demo.glitch.me](https://uskay-portals-demo.glitch.me). Be sure you access with Chrome Canary and turn on the experimental flag!
+1. Enter a URL you want to preview.
+1. The page will then be embedded as a portal element.
+1. Click on the preview.
+1. The preview will be activated after an animation.
+
+<img class="w-screenshot" src="glitch.gif" alt="A gif of using the glitch demo of using Portals">
+
+## Check out the spec
+We are actively discussing [the spec](https://wicg.github.io/portals/) in the Web Incubation Community Group (WICG). To quickly get up to speed, take a look at [the explainer](https://github.com/WICG/portals/blob/master/explainer.md). These are the three important features to familiarize yourself with:
+
+ - [The portal element](https://wicg.github.io/portals/#the-portal-element) … The HTML element itself. The API is very simple. It consists of the `src` attribute, the `activate` function and an interface for messaging (`postMessage`). `activate` takes an optional argument to pass data to the portal upon activation.
```

Comment:
Small nit to switch up the "..." with a colon inside the markdown link. And to do this for each bullet.

Reply:
I will do that! Thank you!